src/dbally/utils/gradio_adapter.py

```diff
@@ -0,0 +1,173 @@
+import sys
+from typing import Dict, List, Optional, Tuple
+
+import gradio
+import pandas as pd
+
+from dbally import BaseStructuredView
+from dbally.collection import Collection
+from dbally.prompts import PromptTemplateError
+from dbally.similarity import SimilarityIndex
+from dbally.utils.errors import NoViewFoundError, UnsupportedQueryError
+from dbally.utils.log_to_file import FileLogger
+
+CONSOLE_FILE_NAME = "console.log"
+sys.stdout = FileLogger(CONSOLE_FILE_NAME)
```

Comment:
we shouldn't override sys.stdout. I think it would be nicer to:

* implement custom parameter to CLIEventHandler to specify where output should be written
* inject dedicated instance of this event_handler into collection to handle gradio output